# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ne/page-up-down.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./src/inputclass.c.orig	2004-02-14 19:05:31.000000000 +0200
+++ ./src/inputclass.c	2004-02-17 19:50:15.000000000 +0200
@@ -108,15 +108,15 @@
 	/* The following bindings are for the terminfo extended codes (see keycodes.h). */
 
 	/* Cursor movement keys */
-	NULL, LINEUP_ABBREV, LINEDOWN_ABBREV, MOVELEFT_ABBREV, MOVERIGHT_ABBREV, MOVEINCUP_ABBREV, MOVEINCDOWN_ABBREV, NEXTPAGE_ABBREV,
-	PREVPAGE_ABBREV, LINEDOWN_ABBREV, LINEUP_ABBREV, NULL, NULL, NULL, NULL, NULL,
+	NULL, LINEUP_ABBREV, LINEDOWN_ABBREV, MOVELEFT_ABBREV, MOVERIGHT_ABBREV, MOVEINCUP_ABBREV, MOVEINCDOWN_ABBREV, PICK(PAGEDOWN_ABBREV, NEXTPAGE_ABBREV),
+	PICK(PAGEUP_ABBREV, PREVPAGE_ABBREV), LINEDOWN_ABBREV, LINEUP_ABBREV, NULL, NULL, NULL, NULL, NULL,
 
 	/* Editing keys */
 	DELETEEOL_ABBREV, NULL, BACKSPACE_ABBREV, DELETELINE_ABBREV, UNDELLINE_ABBREV, DELETECHAR_ABBREV, INSERT_ABBREV, NULL,
 	CLEAR_ABBREV, NULL, NULL, NULL, NULL, NULL, NULL, NULL,
 
 	/* Keypad keys and fake (simulated) menu key */
-	MOVESOF_ABBREV, PREVPAGE_ABBREV, TOGGLESEOL_ABBREV, MOVEEOF_ABBREV, NEXTPAGE_ABBREV, EXEC_ABBREV, NULL, NULL,
+	MOVESOF_ABBREV, PICK(PAGEUP_ABBREV, PREVPAGE_ABBREV), TOGGLESEOL_ABBREV, MOVEEOF_ABBREV, PICK(PAGEDOWN_ABBREV, NEXTPAGE_ABBREV), EXEC_ABBREV, NULL, NULL,
 	NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL,
 
 	NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL,
--- ./src/menu.c.orig	2004-02-14 19:05:30.000000000 +0200
+++ ./src/menu.c	2004-02-17 19:50:15.000000000 +0200
@@ -182,8 +182,8 @@
 		{ "Line Down       ", LINEDOWN_ABBREV },
 PICK(   "Prev Page     ^P", PREVPAGE_ABBREV    , "Page Up       ^P", PAGEUP_ABBREV)
 PICK(   "Next Page     ^N", NEXTPAGE_ABBREV    , "Page Down     ^N", PAGEDOWN_ABBREV)
-PICK(   "Page Up         ", PAGEUP_ABBREV      , "Prev Page       ", PREVPAGE_ABBREV)
-PICK(   "Page Down       ", PAGEDOWN_ABBREV    , "Next Page       ", NEXTPAGE_ABBREV)
+PICK(   "Page Up     PgUp", PAGEUP_ABBREV      , "Prev Page   PgUp", PREVPAGE_ABBREV)
+PICK(   "Page Down   PgDn", PAGEDOWN_ABBREV    , "Next Page   PgDn", NEXTPAGE_ABBREV)
 		{ "Start Of File [A", MOVESOF_ABBREV },
 		{ "End Of File   [E", MOVEEOF_ABBREV },
 		{ "Start Of Line ^A", MOVESOL_ABBREV },
